src/bit_reader.rs

```diff
@@ -34,6 +52,19 @@ extern "C" {
         OutModule: *mut LLVMModuleRef,
     ) -> LLVMBool;
 
+    #[deprecated(since = "3.8", note = "Use LLVMParseIRInContext2")]
+    pub fn LLVMParseIRInContext(
```

Comment:
This is the only function of these that actually seems to exist? Even going back to the 3.8 branch I don't find any mention of the other three functions you've added (nor does it seem to be deprecated).

This should also go in a new `ir_reader` submodule rather than in `bit_reader`, since I try to match the module layout to the structure of the llvm-c include files.

Reply:
Oh, indeed, I may have confused something with our local LLVM changes.
Let's leave it out for now.